src/components/institution/OrganizationBox.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, BoxProps, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { DeleteIconButton } from '../../pages/messages/components/DeleteIconButton';
+import { dataTestId } from '../../utils/dataTestIds';
+import { AffiliationSkeleton } from './AffiliationSkeleton';
+import { OrganizationTitle } from './OrganizationTitle';
+
+export const organizationBoxStyle = {
+  border: '1px solid',
+  borderRadius: '4px',
+  display: 'flex',
+  p: '0.5rem',
+  boxShadow: '0px 3px 3px 0px rgba(0, 0, 0, 0.30)',
+  bgcolor: 'white',
+};
```

Comment:
Her kan vi lage en egen komponent som bare har styling med `styled` (fra `@mui/material`), og den kan brukes som en vanlig komponent uten at vi må sende `sx` som props. Der den brukes kan den fortsatt ta inn `sx` om vi ønsker å overstyre noe

```suggestion
export const StyledOrganizationBox = styled(Box)({
  border: '1px solid',
  borderRadius: '4px',
  display: 'flex',
  padding: '0.5rem',
  boxShadow: '0px 3px 3px 0px rgba(0, 0, 0, 0.30)',
  bgcolor: 'white',
});
```